### DISCUSSION TITLE
Q: Waline 统计文章阅读数时如果并发高时PV计数不准确 || Q: When Waline counts the number of article reads, if the concurrency is high, the PV count will be inaccurate.

### DISCUSSION THREAD

**Discussion by longluo** [walinejs/waline#2486] [Q&A] [answered]
### Describe the bug | 问题描述

如图，百度统计文章阅读数有 **4k+**，但是waline 文章统计阅读次数只有**2k+**，是高并发时统计漏了吗？

![pv](https://github.com/walinejs/waline/assets/3395062/f46665f3-523f-412d-a6ca-7ff945dfe395)

### Website URL | 问题网站

https://www.longluo.me/blog/2024/04/16/2024-alibaba-global-mathematics-competition-qualifying-round/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

As shown in the figure, Baidu counts **4k+** article reads, but Waline counts only **2k+** articles. Is the count missing during high concurrency?

![pv](https://github.com/walinejs/waline/assets/3395062/f46665f3-523f-412d-a6ca-7ff945dfe395)

### Website URL | Problem website

https://www.longluo.me/blog/2024/04/16/2024-alibaba-global-mathematics-competition-qualifying-round/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
这块的实现比较简单，是数据库字段的+1处理，也没有事务，在高并发下是会有问题的。

---
The implementation of this part is relatively simple. It is +1 processing of database fields and there are no transactions. There will be problems under high concurrency.

**Comment by longluo** [comment]
好的，谢谢解答

---
Okay, thank you for your answer
